Library/Formula/libidn.rb

```diff
@@ -23,7 +21,7 @@ def install
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
                           "--disable-csharp"
-    system "make install"
+    system "make", "install"
   end
 
   test do
```

Comment:
If you're feeling extra kind: can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!